storage/cuprate-blockchain/README.md

```diff
@@ -1,5 +1,7 @@
 # Database
-Cuprate's database implementation.
+FIXME: This documentation must be updated and moved to the architecture book.
+
+Cuprate's blockchain implementation.
```

Comment:
This file still uses the name `cuprate-database`.

Since it has to be ported to the architecture book anyway, it will be updated then.